tests/test_sklearn_one_vs_rest_classifier_converter.py

```diff
@@ -388,8 +391,34 @@ def test_ovr_regression_int_ensemble(self):
             model_onnx,
             basename="SklearnOVRRegressionIntEnsemble-Out0",
             allow_failure="StrictVersion(onnxruntime.__version__)"
-            "<= StrictVersion('0.2.1')",
-        )
+            "<= StrictVersion('0.2.1')")
+
+    @unittest.skipIf(not onnx_built_with_ml(),
+                     reason="Requires ONNX-ML extension.")
+    @unittest.skipIf(StrictVersion(ort_version) < StrictVersion("1.2.0"),
+                     reason="fails to load the model")
+    def test_ovr_raw_scores(self):
+        X, y = make_classification(
```

Comment:
We coud use fit_classification_model() and avoid code-repetition.

Reply:
Code repetition is not always bad. In that case, the reader knows exactly what the test does. In my case, fit_classification_model does not expose all the parameters I need. I could expose them or keep it that way.

Reply:
You are using n_samples=100, rest seem to be the same. You may as well use 1000 used in fit_classification_model() or add an additional argument. I don't see why you won't be able to use fit_classification_model() here, unless I am missing something obvious.